server/ServerApp.cpp

```diff
@@ -1479,6 +1480,62 @@ bool ServerApp::IsAvailableName(const std::string& player_name) const {
     return true;
 }
 
+bool ServerApp::IsAuthRequired(const std::string& player_name) {
+    bool result = false;
+    bool success(false);
```

Comment:
pick an initialization style and stick with it

Reply:
Fixed.